bigtop-packages/src/charm/kafka/layer-kafka/tests/01-deploy.py

```diff
@@ -33,9 +33,19 @@ def setUpClass(cls):
         cls.d.configure('openjdk', {'java-type': 'jdk',
                                     'java-major': '8'})
 
-        cls.d.relate('kafka:zookeeper', 'zk:zkclient')
+        try:
```

Comment:
Let's just be backwards compatible without the try/except.  Can we rename the relation to `zookeeper` instead of `zkclient` in the current bigtop zookeeper charm?